.github/workflows/build_v2xhub_docker_image.yml

```diff
@@ -8,38 +8,81 @@ on:
     # Build all tags not containing a / (https://stackoverflow.com/questions/61891328/trigger-github-action-only-on-new-tags  )
     tags:
       - '*'
+
 jobs:
-  v2xhub-docker:
-    # Applied suggested fix: https://github.com/docker/build-push-action/issues/1309
+  build-amd64:
     runs-on: ubuntu-22.04
     steps:
       - name: Checkout
         uses: actions/checkout@v4
         with:
           submodules: recursive
+
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
+
       - name: Log in to Docker Hub
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-      - name: Extract metadata (tags, labels) for Docker
-        id: meta
-        uses: docker/metadata-action@v5
+
+      - name: Build & push AMD64
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64
+          push: true
+          provenance: false
+          tags: usdotfhwaops/v2xhub:amd64-develop
+    
+  build-arm64:
+    runs-on: ubuntu-24.04-arm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
         with:
-          images: usdotfhwaops/v2xhub
-          tags: |
-            type=schedule
-            type=ref,event=branch
-            type=ref,event=tag
-            type=ref,event=pr
-            # set latest tag for default branch
-            type=raw,value=latest,enable={{is_default_branch}}
-      - name: Build and push
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build & push ARM64
         uses: docker/build-push-action@v5
         with:
+          context: .
+          platforms: linux/arm64
           push: true
-          platforms: linux/amd64
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          provenance: false
+          tags: usdotfhwaops/v2xhub:arm64-develop
+
+  manifest:
+    runs-on: ubuntu-22.04
+    needs: [ build-amd64, build-arm64 ]
+    steps:
+      - name: Log in
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Create final multi-arch
+        run: |
+          AMD64_TAG="usdotfhwaops/v2xhub:amd64-develop"
+          ARM64_TAG="usdotfhwaops/v2xhub:arm64-develop"
+          FINAL_TAG="usdotfhwaops/v2xhub:multiarch-develop"
```

Comment:
Update final tag, also tag should not be hard coded to develop since it also needs to work or master. Also this multi-architecture build should happen on pull requests too without actual push (in other words we should confirm that both arm and amd images can be built on pull request

Reply:
Updated final tagging and tested on fork repository